package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@mitodl/smoot-design",
+  "version": "0.0.0",
+  "packageManager": "yarn@4.5.1",
+  "scripts": {
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build --docs",
+    "test": "jest",
+    "typecheck": "tsc --noEmit",
+    "build:esm": "tsc",
+    "build:cjs": "tsc --module commonjs --outDir dist/cjs",
+    "build": "rm -rf dist && rm -f .tsbuildinfo && npm run build:esm && npm run build:cjs",
+    "lint-check": "eslint src/ .storybook/",
+    "lint-fix": "yarn lint-check --fix",
+    "fmt-check": "prettier --ignore-path .gitignore --ignore-path .prettierignore --check .",
+    "fmt-fix": "prettier --ignore-path .gitignore --ignore-path .prettierignore --write ."
```

Comment:
Are these script names etched anywhere in OL standards? I use them a lot but often can't remember them. Suggest e.g.:

```
"lint": "eslint src/ .storybook/",
"lint:fix": "yarn lint --fix",
"prettier": "prettier --ignore-path .gitignore --ignore-path .prettierignore --check .",
"prettier:fix": "yarn prettier --write"
```

Reply:
`fmt` and `lint` are fairly standard across our projects, though often the combo is `fmt, fmt-check` rather than `fmt-fix, fm-check`. I used `fmt-fix` for this and Learn b/c it seems more explicit.

In general, these should run automatically via `pre-commit` (either on git commit, or if you trigger them manually, e.g., via `pre-commit run --all`).